example/android/app/src/main/java/com/example/reactnativenetworkclient/MainApplication.java

```diff
@@ -49,6 +51,11 @@ public void onCreate() {
     super.onCreate();
     SoLoader.init(this, /* native exopackage */ false);
     initializeFlipper(this, getReactNativeHost().getReactInstanceManager()); // Remove this line if you don't want Flipper enabled
+
+    // Tells React Native to use our RCTOkHttpClientFactory which builds an OKHttpClient
+    // with a cookie jar defined in APIClientModule and an interceptor to intercept all
+    // requests that originate from React Native's OKHttpClient
+    OkHttpClientProvider.setOkHttpClientFactory(new RCTOkHttpClientFactory());
```

Comment:
This is 🔥!

So will this override anywhere OkHttp is now called? Across the packages / code?